tests/integration/test_vulnerability_detector/test_scan_results/data/custom_nvd_feed.json

```diff
@@ -1122,6 +1122,528 @@
     },
     "publishedDate" : "2020-01-14T23:15Z",
     "lastModifiedDate" : "2020-01-22T12:26Z"
+  },

```

Comment:
This feed file is shared by other tests, so make sure it doesn't break any other tests :D

Reply:
Of course.